src/routes/config.js

```diff
@@ -0,0 +1,26 @@
+import { bind, getContainer } from '@globality/nodule-config';
+import { OK } from 'http-status-codes';
+import { cloneDeep, has, set } from 'lodash';
+
+export default function configRoute(req, res) {
+    const { config } = getContainer();
+
+    if (!has(config, 'secrets')) {
+        return res.status(OK).json('Config sharing disabled if no secrets are labeled');
+    }
+
+    // Set any fields that are secrets to undefined
+    const clonedConfig = cloneDeep(config);
+    clonedConfig.secrets.forEach((secret) => {
+        if (has(clonedConfig, secret)) {
+            set(clonedConfig, secret, undefined);
```

Comment:
Would this also expose something like:
```
{
   secret-key: undefined
}
```
Because I think we remove those from the config as well.